test/integration/local_plugins/conftest.py

```diff
@@ -1,6 +1,9 @@
 import pytest
 
-from helper import setup_easy_client, RequestGenerator
+try:
+    from helper import setup_easy_client, RequestGenerator
+except:
+    from ..helper import setup_easy_client, RequestGenerator
```

Comment:
I saw this in another place as well... why is this necessary? It it because the parent and child gardens are in different "levels?" could we move the parent down a level so this wouldn't be needed?

Reply:
Honestly, I didn't try having the `..helper` in the tests in github. I just moved the folder and hoped for the best. I can try not moving the folder and seeing what happens.

Reply:
So, I went back and tested this out in a couple different combos. If I want to be able to run the tests in PyCharm I need `..helper`. If I run them through the Make file, then `helper` is all that is needed. So this is more a quality of life thing for people using an IDE.